### ISSUE TITLE
Unit test

### ISSUE THREAD

**Issue by YuMiao329** [YuMiao329/test_repository#3]
[https://github.com/YuMiao329/test_repository/blob/04938de876bd9582c8bda7e8b3447db114878191/test_find_y.py](url)

**Comment by dward2** [comment]
Nate, when I try to run your tests, the test fail with a `ModuleNotFoundError:  No module named 'on_plane'`.  Can you fix that?

**Comment by YuMiao329** [comment]
> Nate, when I try to run your tests, the test fail with a `ModuleNotFoundError: No module named 'on_plane'`. Can you fix that?

Yes, Dr. Ward, I have fixed that problem. It should be okay now. Let me know if there is any other problem.

**Comment by dward2** [comment]
Excellent.  Looks good.  Thanks.
